src/maptrait.rs

```diff
@@ -3,9 +3,14 @@ use std::str::FromStr;
 use std::hash::Hash; 
 use std::collections::HashMap; 
 
+
+///Trait for custom implementation of formatting 
```

Comment:
I think replace this with "Represents a generic Map. Implemented for HashMap and (optionally) serde's Map"

Reply:
Done